account_billing/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    'name': "Billing Process",
```

Comment:
Please use single quote on all.
```suggestion
    'name': 'Billing Process',
```